### PR TITLE
Minor change regarding URLs

### PR DESCRIPTION
**Pull request by bobvanluijt**
Goals:

- Made two minor changes, set the correct URLs in the docs

### CODE REVIEW

**Comment by davidbp** [comment]
Just tried `https://www.weaviate.io` in my browser and did not work. Are you sure you want to put this URL?

File: docs/advanced/document-store/weaviate.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (weaviate)=
 # Weaviate
 
-One can use [Weaviate](https://www.semi.technology) as the document store for DocumentArray. It is useful when one wants to have faster Document retrieval on embeddings, i.e. `.match()`, `.find()`.
+One can use [Weaviate](https://www.weaviate.io) as the document store for DocumentArray. It is useful when one wants to have faster Document retrieval on embeddings, i.e. `.match()`, `.find()`.
```
Comment (alaeddine-13) [inline docs/advanced/document-store/weaviate.md:4]:
```suggestion
One can use [Weaviate](https://weaviate.io) as the document store for DocumentArray. It is useful when one wants to have faster Document retrieval on embeddings, i.e. `.match()`, `.find()`.
```
I guess this is the correct link right ?

**Comment by bobvanluijt** [comment]
> Just tried `https://www.weaviate.io` in my browser and did not work. Are you sure you want to put this URL?

It's `https://weaviate.io` 👍

**Comment by bobvanluijt** [comment]
O sorry, that was my mistake 🙈 
I believe it's all good now

**Comment by alaeddine-13** [comment]
I'm closing this PR and fixing commit linting in another PR

**Pull request closed without merging**